.coveragerc

```diff
@@ -0,0 +1,3 @@
+[report]
+omit =
+    */tests/*
```

Comment:
I don't think there should be a line break in this, should it?

Reply:
omit is multistring parameter. there are line breaks in example configuration: https://coverage.readthedocs.io/en/v4.5.x/source.html#execution